sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -38,6 +38,11 @@ _Static_assert(
     RSTMGR_PARAM_NUMSWRESETS <= 32,
     "Reset Enable and Control registers span across multiple registers!");
 
+// Make sure that the public alert info crash dump size matches the HW.
+_Static_assert(
+    DIF_RSTMGR_ALERT_INFO_MAX_SIZE == RSTMGR_ALERT_INFO_CTRL_INDEX_MASK,
```

Comment:
I'm a little bit confused because both `INDEX` and `CNT_AVAIL` are 4 bits wide. Normally I would say size should be mask + 1, but it looks like max size is indeed 15 in this case and therefore max index is 14, which is a little bit odd. Apologies if this was discussed before but just wanted to make sure.